.travis.yml

```diff
@@ -37,6 +38,7 @@ matrix:
         # restart jackrabbit after each suite see: https://github.com/sulu-io/sulu/issues/2137
         - TEST_FLAGS="--jackrabbit-restart"
         - SYMFONY_DEPRECATIONS_HELPER="weak"
+        - PHPSTAN=true
```

Comment:
Is there a reason you added this to PHP 7.1? If we do it with the latest version it might be a bit faster :-) Or does phpstan also check under which version it is ran, and it should therefore be with the lowest one?